src/client/driver/driver-link/child.js

```diff
@@ -76,7 +76,7 @@ export default class ChildDriverLink {
             });
     }
 
-    confirmConnectionEstablished (requestMsgId) {
+    confirmMessage (requestMsgId) {
```

Comment:
I will create a separate function `sendConfirmationMessage` and call it in appropriate places.
I prefer the full variant 'sendConfirmationMessage' instead of 'sendConfirmMessage'.